package.json

```diff
@@ -38,61 +38,63 @@
   },
   "dependencies": {
     "ajv": "^5.2.2",
+    "codecov": "^3.8.1",
     "compassql": "0.20.2",
     "font-awesome": "^4.7.0",
-    "font-awesome-sass-loader": "^1.0.3",
-    "isomorphic-fetch": "^2.2.1",
-    "moment": "^2.18.1",
-    "normalize-scss": "^6.0.0",
-    "papaparse": "^4.3.5",
+    "font-awesome-sass-loader": "^2.0.1",
+    "isomorphic-fetch": "^3.0.0",
+    "moment": "^2.29.1",
```

Comment:
Can we get rid of this? Moment was useful a few years ago but there are lighter alternatives today.

Reply:
What should I replace this moment with? It is marked as a peer dependency of react-moment.

Reply:
Leave it for now then but we may not need react moment anymore. We can just use the browser APIs for time.